### ISSUE TITLE
How to turn off the function of 'lrf' and 'warmup'?

### ISSUE THREAD

**Issue by whh722** [ultralytics/yolov5#2880]
## ❔Question
How to turn off the function of 'lrf' and 'warmup'?How to adjust 'lrf','warmup_epochs','warmup_momentum'and 'warmup_bias_lr' in '.hpy' files?

## Additional context
I mean how can i keep the learning rate in 'lr0'.
![UQYF7`9A EE}49 (M_B0XIU](https://user-images.githubusercontent.com/78618616/115591530-c1b51980-a304-11eb-9eea-13b0a95430ce.png)

**Comment by glenn-jocher** [comment]
@whh722 you can modify any hyperparameters you want in your hyp file, i.e.:
https://github.com/ultralytics/yolov5/blob/5f7d39fede4de8af98472bd009c63c3a86568e2d/data/hyp.scratch.yaml#L1-L33
